docs/fundamentals/executor/instrumenting-executor.md

```diff
@@ -0,0 +1,272 @@
+(instrumenting-executor)=
+# Instrumentation
+
+Instrumentation consists of [OpenTelemetry](https://opentelemetry.io) Tracing and Metrics. Each features can be enabled independently and allow you to collect request-level and application level metrics for analyzing an Executor's real-time behavior. 
+
+This section documents creating custom traces and metrics.
+
+```{admonition} Full details on Instrumentation
+:class: seealso
+This section describes **custom** tracing spans. To use the Executor's default tracing, refer to {ref}`the Flow Instrumentation <instrumenting-flow>` section.
+```
+
+```{hint}
+The OpenTelemetry collector backend setup is described in the {ref}`OpenTelemetry Setup <opentelemetry>` section.
+```
+
+```{hint}
```

Comment:
```suggestion
```{caution}
```

It would be nice if you could spread these boxes around a bit, it is a bit jarring to have three boxes next to each other.